src/System.Diagnostics.Process/src/System.Diagnostics.Process.builds

```diff
@@ -11,6 +11,9 @@
     <Project Include="System.Diagnostics.Process.csproj">
       <OSGroup>Windows_NT</OSGroup>
     </Project>
+    <Project Include="System.Diagnostics.Process.csproj">
+      <TargetGroup>net461</TargetGroup>
+    </Project>
     <Project Include="facade\System.Diagnostics.Process.csproj">
```

Comment:
Given the churn in this project I think it make more sense to include the net461 facade in the existing facade project.